ArduCopter/mode_guided.cpp

```diff
@@ -247,11 +253,14 @@ bool ModeGuided::set_destination(const Vector3f& destination, bool use_yaw, floa
     // set yaw state
     set_yaw_state(use_yaw, yaw_cd, use_yaw_rate, yaw_rate_cds, relative_yaw);
 
-    // no need to check return status because terrain data is not used
-    wp_nav->set_wp_destination(destination, terrain_alt);
+    // record velocity target
```

Comment:
I guess this comment is incorrect because it's not recording a velocity target but instead a position target.